service/src/main/kotlin/app/cash/backfila/service/persistence/DbRegisteredBackfill.kt

```diff
@@ -66,13 +66,22 @@ class DbRegisteredBackfill() : DbUnsharded<DbRegisteredBackfill>, DbTimestampedE
   @Column
   var requires_approval: Boolean = false
 
+  /**
+   * Notes when this backfill should no longer be useful and the code should be deleted. This will
```

Comment:
"will be" is a strange comment to have. If you change the comment when implementing the reminders then its all good

Reply:
Good point. Fixed